bigquery/google/cloud/bigquery/_helpers.py

```diff
@@ -90,12 +90,15 @@ def _timestamp_query_param_from_json(value, field):
 
     Args:
         value (str): The timestamp.
-        field (.SchemaField): The field corresponding to the value.
+
+        field (google.cloud.bigquery.schema.SchemaField):
```

Comment:
If I'm not too late to the party - it would make sense to cross-link classes in the docs (applies to multiple places).

```suggestion
        field (:class:`~google.cloud.bigquery.schema.SchemaField`):
```

Reply:
Well, in fact, Tim had been observing the process of writting the script (which made this changes), and it was decided to remove `:class:` tags from type definitions, but to keep them in arg descriptions
![Безымянный](https://user-images.githubusercontent.com/26030071/67616427-2dd2c200-f7e1-11e9-8811-91ee2e3821a2.png)

Reply:
I see, thanks, didn't know that.